qiling/os/uefi/utils.py

```diff
@@ -18,4 +20,20 @@ def check_and_notify_protocols(ql):
         ql.reg.rcx = notify_context
         ql.reg.arch_pc = notify_func
         return True
-    return False
\ No newline at end of file
+    return False
+
+def write_int32(ql, address, num):
+    if ql.archendian == QL_ENDIAN.EL:
+        ql.mem.write(address, struct.pack('<I',(num)))
+    else:
+        ql.mem.write(address, struct.pack('>I',(num)))
```

Comment:
please put blank lines between functions (double-lines is even better)

Reply:
I'm sure I did, maybe my editor isn't setup correctly and removed.

Reply:
Done.